model/dce-time.cc

```diff
@@ -42,6 +42,13 @@ struct tm * dce_localtime (const time_t *timep)
   return localtime_r (timep, &Current ()->process->struct_tm);
 }
 
+struct tm * dce_localtime_r (const time_t *timep, struct tm *buf)
+{
+  setenv ("TZ", "UTC", 1);
+  tzset ();
```

Comment:
I'm just wondering; does this return different result between `dce_localtime` and `dce_localtime_r` ? It would be nice if we can do similar.

Reply:
It does return a different result. Simply redirecting `dce_localtime_r` to `dce_localtime` the logs in FRR start at `2009/12/31 22:00:00`. With that implementation the logs start at `2010/01/01 00:00:00` which I found clearer. 

Nevertheless it's a matter of preference and it could just redirect to `dce_localtime_r`.